docs/HeliumParamSpecs.md

```diff
@@ -0,0 +1,295 @@
+# Helium Parameter Specs Documentation
+
+## System Overview
+
+Helium is WebAPI reference application designed to "fork and code" in c#, java, typescript languages with the following features:
+
+* Securely build, deploy and run an App Service (Web App for Containers) application
+* Use Managed Identity to securely access resources
```

Comment:
Use "-" for unordered lists in markdown, per https://github.com/retaildevcrews/helium/blob/master/docs/EngineeringPractices.md

Reply:
I removed this as well and provided a reference to main Helium readme as it's repetitive and should point to one source.